bridge_adaptivity/config/settings/base.py

```diff
@@ -158,3 +159,5 @@
     ),
     'TEST_REQUEST_DEFAULT_FORMAT': 'json',
 }
+
+FONTAWESOME_CSS_URL = 'https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css'
```

Comment:
as for me the name of this variable is pretty clear to understand. What should I write here?
`# This is Font awesome css URL`? as for me it would look like duplication with var name.